players/api.py

```diff
@@ -1,24 +1,38 @@
 from rest_framework import viewsets
+from rest_framework import permissions
+from rest_framework.response import Response
+from players.models import Player
+from players.serializers import UserDataSerializer, RegisterSerializer
 # Test test
 # Another test
 # fix
 
+
 class PlayerViewSet(viewsets.GenericViewSet):
-    permission_classes = []
-    authentication_classes = []
+    queryset = Player.objects.all()
+    serializer_class = UserDataSerializer
+    permission_classes = (permissions.AllowAny, )
+    # authentication_classes = []
 
-    def list(self, request):
+    def create(self, request, *args, **kwargs):
+        """Creates an user from the Django model"""
+        serializer = RegisterSerializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+        serializer.save()
 
-        pass
+        return Response({"Message": "Usuario creado"})
 
-    def retrieve(self, request, pk):
-        pass
+    def list(self, request):
+        all_players = Player.objects.all()
+        serializer = UserDataSerializer(all_players, many=True)
```

Comment:
The pull request must only include only the code for the issue that is resolving, this code must not be here